sc/src/main/webapp/WEB-INF/jsp/tiles/views/home/home-panels.jsp

```diff
@@ -0,0 +1,42 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
```

Comment:
JSPs inside the directory `views/home` are HTML fragments for the home page. You shouldn’t include any `<html>`, `<head>` or `<body>` tags (have a look at the other fragments in this directory). Also, remove references to CSS files that are included in the base template (`base.jsp`). Also, notice that you’re adding a style from Expression Atlas, and you should only use the styles from Single Cell Expression Atlas. If there are styles missing add them accordingly (be careful about copy & paste and add only and strictly what you need).